pkg/validation/internal/testdata/dockerfile_v4_8/bundle_without_label.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM scratch
```

Comment:
nit: it would be nice if we could template these and stamp generate our fixtures at runtime instead of keeping a bunch of near-duplicates around

Reply:
That would be nicer by sure. We need to clean up this implementation.